exporter/node.go

```diff
@@ -104,9 +103,26 @@ func New(opts Options) Exporter {
 		ibftSyncEnabled: opts.IbftSyncEnabled,
 	}
 
+	if err := e.init(opts); err != nil {
+		e.logger.Fatal("failed to init", zap.Error(err))
+	}
+
 	return &e
 }
 
+func (exp *exporter) init(opts Options) error {
+	if opts.CleanRegistryData {
```

Comment:
@amir-blox why not put the delete operation in the cli package?

Reply:
The delete is not "generic" - it uses some prefix, meaning we need to perform this from the actual `collection`/`storage` where the prefix is defined

Reply:
@amir-blox it can cause issues in the future if we need something else deleted and the logic is spread around in multiple places.
I guess for now it will do